src/layouts/adminlayout.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { Outlet } from "react-router-dom";
+import Sidebar from "../components/navigation/sidebar.jsx";
+
+const LayoutAdmin = () => {
```

Comment:
💊 파일명과 컴포넌트 명을 통일해주시면 좋을 것 같습니다!
```suggestion
const AdminLayout = () => {
```